common/containers/Tabs/ENS/components/generalInfoPanel/helperStatelessFunctionals.jsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import type { ABlankNooperProps } from './types';
+
+export const ABlankNooper = ({
+  content, //eslint-disable-line
+  href //eslint-disable-line
+}): ABlankNooperProps =>
+  <a target="_blank" rel="noopener" href={href}>
+    {content}
+  </a>;
```

Comment:
I really like this component, and would like to see it in the shared components folder. My only suggestion is to use the `children` props so it behaves more like a traditional link. And maybe a more intent-based name, like "NewTabLink" or something.

Reply:
Right, mainly used content for a short-hand way of setting the HTML inside for one-off links. I can make it so it can accept `children || content`. Also, originally it would accept `{children, ...rest}` as props and then you would be able to over-ride any defaults/add any properties to `<a/>` but flow was throwing a fit  on how `rest` did not contain certain properties.

Reply:
Yes, the shared components is in `/common/components`